databroker_browser/qt/_core.py

```diff
@@ -326,11 +343,15 @@ def __init__(self, db, fig_dispatch, text_dispatch, result_dispatch):
         
         layout = QtWidgets.QVBoxLayout()
         sublayout = QtWidgets.QHBoxLayout()
+        results_pane = QtWidgets.QVBoxLayout()
+        results_pane.addWidget(self._results_summary)
+        results_pane.addWidget(self._results)
         layout.addWidget(self._search_bar)
         layout.addLayout(sublayout)
-        sublayout.addWidget(self._results)
+        sublayout.addLayout(results_pane)
         sublayout.addWidget(self._hvw.widget)
         self.widget.setLayout(layout)
+        self.search()
```

Comment:
This initializes the search results with the last 100 (or the last `self.max_results`) headers. Formerly the app loaded blank which was necessary (we didn't have lazy-loading yet!) but confusing to users.